elide-graphql/src/test/java/example/Author.java

```diff
@@ -10,6 +10,8 @@
 import com.yahoo.elide.annotation.Include;
 import com.yahoo.elide.annotation.SharePermission;
 
+import lombok.Builder;
```

Comment:
Is this unused?

Reply:
This is used in a javadoc:

```
/**
 * Model for authors.
 * <p>
 * <b>CAUTION: DO NOT DECORATE IT WITH {@link Builder}, which hides its no-args constructor. This will result in
 * runtime error at places such as {@code entityClass.newInstance();}</b>
 */
```

Reply:
I mean can you remove the import since it is unused?